cli/utils/api.go

```diff
@@ -222,6 +222,22 @@ func (resourceClient ResourceClient) List(ctx context.Context, listArgs ListArgs
 		return nil, fmt.Errorf("could not send request: %w", err)
 	}
 
+	if resp.StatusCode == http.StatusBadRequest {
```

Comment:
Definitely not the best but we can find a way to include better error handling across the CLI in the future!